### ISSUE TITLE
[autofix] Automatic fixes replace single line comment with multi line comment when on same line as @require

### ISSUE THREAD

**Issue by Cactusbone** [stylus/stylelint-stylus#16]
Hello, using `--fix`, I've found some unexpected changes:

for example, `@import nib //for mixins` becomes `@import nib /*for mixins*/`

Is it a bug or is there an explanation somewhere ?

I'm using this config (still in the middle of fixing issues)

```
module.exports = {
    extends: [
        'stylelint-stylus/standard',
    ],
    rules: {
        'stylus/indentation': 4,
        'stylus/at-extend-style': null,
        'stylus/at-rule-empty-line-before': null,
        'stylus/block-opening-brace-space-before': null,
        'stylus/color-hex-case': null,
        'stylus/declaration-colon': null,
        'stylus/hash-object-property-comma': null,
        'stylus/media-feature-colon': null,
        'stylus/no-at-require': null,
        'stylus/number-leading-zero': null,
        'stylus/property-no-unknown': null,
        'stylus/pythonic': null,
        'stylus/selector-list-comma': null,
        'stylus/selector-list-comma-newline-after': null,
        'stylus/selector-type-no-unknown': null,
        'stylus/semicolon': null,
        'stylus/single-line-comment': null,
    },
};

```

**Comment by ota-meshi** [comment]
Thanks for the many issue reports. I will check them one by one when I have time.

**Comment by ota-meshi** [comment]
Fixed in [postcss-styl v0.12.3](https://github.com/stylus/postcss-styl/releases/tag/v0.12.3). Please update postcss-style.
